curvefs/test/mds/fs_manager_test.cpp

```diff
@@ -635,132 +869,26 @@ TEST_F(FSManagerTest, backgroud_thread_test) {
-    ASSERT_EQ(volumeFsInfo1.rootinodeid(), ROOTINODEID);
-    ASSERT_EQ(volumeFsInfo1.capacity(), volumeSize);
-    ASSERT_EQ(volumeFsInfo1.blocksize(), blockSize);
-    ASSERT_EQ(volumeFsInfo1.mountnum(), 0);
-    ASSERT_EQ(volumeFsInfo1.fstype(), ::curvefs::common::FSType::TYPE_VOLUME);
-
-    // create s3 test
-    std::string fsName2 = "fs2";
-    curvefs::common::S3Info s3Info;
-    FsInfo s3FsInfo;
-    s3Info.set_ak("ak");
-    s3Info.set_sk("sk");
-    s3Info.set_endpoint("endpoint");
-    s3Info.set_bucketname("bucketname");
-    s3Info.set_blocksize(4096);
-    s3Info.set_chunksize(4096);
-    CreateRootInodeResponse response2;
-    FsDetail detail2;
-    detail2.set_allocated_s3info(new S3Info(s3Info));
-
-    // create s3 fs ok
-    EXPECT_CALL(*topoManager_, CreatePartitionsAndGetMinPartition(_, _))
-        .WillOnce(Return(TopoStatusCode::TOPO_OK));
-    EXPECT_CALL(*topoManager_, GetCopysetMembers(_, _, _))
-        .WillOnce(
-            DoAll(SetArgPointee<2>(addrs), Return(TopoStatusCode::TOPO_OK)));
-    EXPECT_CALL(mockCliService2_, GetLeader(_, _, _, _))
-        .WillOnce(
-            DoAll(SetArgPointee<2>(getLeaderResponse),
-                  Invoke(RpcService<GetLeaderRequest2, GetLeaderResponse2>)));
-
-    response.set_statuscode(MetaStatusCode::OK);
-    EXPECT_CALL(mockMetaserverService_, CreateRootInode(_, _, _, _))
-        .WillOnce(DoAll(
-            SetArgPointee<2>(response),
-            Invoke(
-                RpcService<CreateRootInodeRequest, CreateRootInodeResponse>)));
-    EXPECT_CALL(*s3Adapter_, BucketExist()).WillOnce(Return(true));
-
-    req.set_fsname(fsName2);
-    req.set_fstype(FSType::TYPE_S3);
-    req.set_allocated_fsdetail(new FsDetail(detail2));
-    ret = fsManager_->CreateFs(&req, &s3FsInfo);
-    ASSERT_EQ(ret, FSStatusCode::OK);
-    ASSERT_EQ(s3FsInfo.fsid(), 1);
-    ASSERT_EQ(s3FsInfo.fsname(), fsName2);
-    ASSERT_EQ(s3FsInfo.status(), FsStatus::INITED);
-    ASSERT_EQ(s3FsInfo.rootinodeid(), ROOTINODEID);
-    ASSERT_EQ(s3FsInfo.capacity(), (uint64_t)100 * 1024 * 1024 * 1024);
-    ASSERT_EQ(s3FsInfo.blocksize(), blockSize);
-    ASSERT_EQ(s3FsInfo.mountnum(), 0);
-    ASSERT_EQ(s3FsInfo.fstype(), FSType::TYPE_S3);
+    std::set<std::string> addrs { addr_ };
+    FSStatusCode ret;
 
     // TEST GetFsInfo
     FsInfo fsInfo1;
     ret = fsManager_->GetFsInfo(fsName1, &fsInfo1);
-    ASSERT_EQ(ret, FSStatusCode::OK);
-    ASSERT_TRUE(CompareVolumeFs(fsInfo1, volumeFsInfo1));
 
     FsInfo fsInfo2;
     ret = fsManager_->GetFsInfo(fsName2, &fsInfo2);
-    ASSERT_EQ(ret, FSStatusCode::OK);
-    ASSERT_TRUE(CompareS3Fs(fsInfo2, s3FsInfo));
 
     // TEST DeleteFs, delete fs1
-    std::list<PartitionInfo> list;
-    std::list<PartitionInfo> list2;
-    std::list<PartitionInfo> list3;
+    std::list<PartitionInfo> empty_list;
+    std::list<PartitionInfo> one_readable_writeabe_partition_list;
+    std::list<PartitionInfo> deleting_list;
```

Comment:
camelCase for variable name, `emptyList`

Reply:
renamed